### DISCUSSION TITLE
turn off consent

### DISCUSSION THREAD

**Discussion by JeroenMX** [simpleidserver/SimpleIdServer#337] [General]
Hi,

Is there a way to skip the consent screen when logging in? There doesn't seem to be something like a property 'RequireConsent' on a client that can be set to false. For my project i need to skip this screen.

Thanks,
Jeroen

**Comment by simpleidserver** [comment]
Hello,

At the moment, it is not possible to skip the consent. I create a ticket to support this feature (#338)

KR,

SID

**Comment by simpleidserver** [comment]
The feature is implemented in the `master` branch.

KR,

SID
